docs/developer-docs/multi-chain/faq/user-faq.mdx

```diff
@@ -12,11 +12,30 @@ The following questions and answers are from the POV of a user using consumer ap
 
 For questions from the POV of a developer who wants to know things such as "how does a canister send ckBTC?" or how to write code for smart contracts to hold, send, or receive, please see the other FAQs.
 
+## General multi-chain questions
+
+### What does it mean that the Internet Computer is multi-chain?
```

Comment:
To me, the wording seems like a conceptual mismatch. How can ICP, being one blockchain network, be "multi-chain" as a network? I understand that a dapp may be a "multi-chain dapp" (making use of several chains), and ICP supporting multi-chain dapps, but not ICP *being* multi-chain.